web/viewer.js

```diff
@@ -36,6 +36,13 @@ window.PDFViewerApplication = PDFViewerApplication;
 window.PDFViewerApplicationConstants = AppConstants;
 window.PDFViewerApplicationOptions = AppOptions;
 
+if (AppOptions.get("enableStampEditor") === true) {
+  const editorStampButton = document.getElementById("editorStamp");
+  if (editorStampButton) {
+    editorStampButton.hidden = false;
+  }
+}
+
```

Comment:
Please move into the following code instead https://github.com/mozilla/pdf.js/blob/8b50836d76bbfd33f203887f4ea431e23f0344e4/web/app.js#L561-L562